### ISSUE TITLE
Restarting loop

### ISSUE THREAD

**Issue by WDPK837** [realies/soulseek-docker#27]
Hi all, trying to install this container from the latest build onto my Synology DS920. I can install it from the Docker app on DSM and configure it, but then when I try to run the software it goes into a restart loop, throwing an "exec format" error. I've tried reconfiguring and reinstalling with no luck.

**Comment by jt196** [comment]
Same here - no logs, nuffink. Synology DS918+

**Comment by jt196** [comment]
Figured it. My problem was permissions. I had mounted my music folder without changing group/user ID to the container's.

**Comment by jt196** [comment]
Just a quick update, this was my Music folder share interfering with the set up. I'm not to hot on logging but there's not much info/feedback about this - would be great if the log had some indication that permissions weren't correct on one of the mounted folders. 

My folders:

```
            - '/volume1/docker/soulseek/data:/data/.SoulseekQt'
            - '/volume1/downloads/soulseek:/data/Soulseek Downloads'
            - '/volume1/docker/soulseek/logs:/data/Soulseek Chat Logs'
            - '/volume1/music/Plex:/data/Music'
```
My permissions on the docker and downloads folders were set correctly already. I just changed the music folder permissions:
```
chown -R 1029:100 Plex
chmod -R 755 Plex
```
Interestingly, I tried mounting the Music folder with a read only tag, but it wasn't having any of it.
